_signal_processing/signal_utils.py

```diff
@@ -80,3 +83,8 @@ def write_wav(f, sample_freq, x):
     from scipy.io.wavfile import write
 
     write(f, sample_freq, x)
+
+
+def ok_to_store_file(path: str) -> bool:
```

Comment:
I would rename function starting with verbs like `is`, `has`, `can` and etc.
Could you rename it to `can_store_file`